packages/react-components/src/utils/type.tsx

```diff
@@ -0,0 +1,10 @@
+export type Dispatch<T> = (x?: T) => any
+
+export declare type IHandler = {
```

Comment:
Don’t declare a new one. It is already been declared in @aemforms/af-react-renderer.
import { Handlers } from '@aemforms/af-react-renderer';

Reply:
Implemented!